PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
The 'ABC's aren't solid in my mind, but this could be `Sequence`? (__getitem__, __len__)